internal/k8shandler/forwarding.go

```diff
@@ -12,17 +13,29 @@ import (
 	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
 	"github.com/openshift/cluster-logging-operator/internal/constants"
 	"github.com/openshift/cluster-logging-operator/internal/generator"
-	"github.com/openshift/cluster-logging-operator/internal/tls"
+
 	corev1 "k8s.io/api/core/v1"
 )
 
-// useOldRemoteSyslogPlugin checks if old plugin (docebo/fluent-plugin-remote-syslog) is to be used for sending syslog or new plugin (dlackty/fluent-plugin-remote_syslog) is to be used
-func (clusterRequest *ClusterLoggingRequest) useOldRemoteSyslogPlugin() bool {
-	if clusterRequest.ForwarderRequest == nil {
-		return false
+// EvaluateAnnotationsForEnabledCapabilities populates generator options with capabilities enabled by the ClusterLogForwarder
+func EvaluateAnnotationsForEnabledCapabilities(forwarder *logging.ClusterLogForwarder, options generator.Options) {
+	if forwarder == nil {
+		return
+	}
+	for key, value := range forwarder.Annotations {
+		switch key {
+		case constants.PreviewTLSSecurityProfile:
```

Comment:
Do nothing here, maybe just remove this `case`

Reply:
It is doing something for constants.PreviewTLSSecurityProfile, which is the same thing as for constants.UseOldRemoteSyslogPlugin. This is the same as
```
switch key {
case constants.PreviewTLSSecurityProfile, constants.UseOldRemoteSyslogPlugin:
...
```